test/run-souffle-tests.py

```diff
@@ -44,6 +44,7 @@
     "cellular_automata", # issue 198 - order of clauses
     "comp-override2", # nested component declaration
     "components1",    # improper component nesting
+    "float_operations",
```

Comment:
I will try to add float and double to ddlog.
not sure yet about literals, but at least the types.

Reply:
That'd be awesome, thanks. I can help with Rust bindings. Here's a relevant thread:
https://internals.rust-lang.org/t/pre-rfc-dealing-with-broken-floating-point/2673

Sounds like different CPUs can produce different results for the same floating operations.  This is going to be a problem for D3log. It also sounds like this should not be an issue for most modern desktops/servers, so maybe this is nothing to worry about.